src/netaddress.cpp

```diff
@@ -179,12 +179,6 @@ bool CNetAddr::IsLocal() const
    return false;
 }
 
-bool CNetAddr::IsMulticast() const
```

Comment:
ACK on this one. Cannot find when this was *ever* used, and I don't think we need this now or on the forseeable future.